Formula/h/heppdt2.rb

```diff
@@ -1,8 +1,8 @@
 class Heppdt2 < Formula
   desc "Translation for particle ID's to and from various MC generators and PDG standard"
   homepage "https://cdcvs.fnal.gov/redmine/projects/heppdt/wiki"
```

Comment:
As below but so this doesn't get lost: this link requires a login. We need a new homepage here.